app/Helpers/database/set-claim.php

```diff
@@ -455,26 +457,27 @@ function updateClaim(int $claimID, int $claimType, int $setType, int $status, in
  */
 function getExpiringClaim(string $user): array
 {
-    sanitize_sql_inputs($user);
-
-    $query = "
-        SELECT
-            COALESCE(SUM(CASE WHEN TIMESTAMPDIFF(MINUTE, NOW(), sc.Finished) <= 0 THEN 1 ELSE 0 END), 0) AS Expired,
-            COALESCE(SUM(CASE WHEN TIMESTAMPDIFF(MINUTE, NOW(), sc.Finished) BETWEEN 0 AND 10080 THEN 1 ELSE 0 END), 0) AS Expiring
-        FROM
-            SetClaim AS sc
-        WHERE
-            sc.User = '$user'
-            AND sc.Status IN (" . ClaimStatus::Active . ',' . ClaimStatus::InReview . ")
-            AND sc.Special != " . ClaimSpecial::ScheduledRelease;
-
-    $dbResult = s_mysql_query($query);
+    if (empty($user)) {
```

Comment:
here's an opportunity to rename this variable to `$username` to better describe what this is and not run into ambiguity with `$user` referencing a full user model